libs/platform/src/lib/approval-flow/approval-flow-add-node/approval-flow-add-node.component.ts

```diff
@@ -226,6 +217,8 @@ export class ApprovalFlowAddNodeComponent implements OnInit, OnDestroy {
     ngOnDestroy(): void {
         this._viewChangeSub.unsubscribe();
         this._dataSourceSub.unsubscribe();
+        this._data.userDataSource.close();
+        this._data.teamDataSource.close();
```

Comment:
IMHO if we offer the ApprovalFlowDataSource in the doc as one ready to go and actually are using it in the approval-flow component we need to have this method implemented so it does the job.
Otherwise it's not clear and a user might be confused.
Could you please add a minimal implementation for `dataSource.close()` method so at least it closes a stream that was opened via `dataSource.open()`. I suppose this change should be done in ComboboxDataSource class.

Reply:
sure, done

Reply:
thank you